help/en/package/jmri/jmrit/logixng/LogixImport.shtml

```diff
@@ -25,19 +25,33 @@
         <div style="margin-left: 2em">
         <a href="images/ImportLogix.png">
         <img src="images/ImportLogix.png"
-        alt="Import Logix to LogixNG" width="610" height="515" >
+        alt="Import Logix to LogixNG" width="610" height="586" >
         </a>
         </div>
 
         <h2>Import Options</h2>
 
           <p>Most of the options have not been implemented yet.</p>
 
+          <h4>Import system Logixs also?</h4>
+
           <p>The <strong>Import system Logixs also?</strong> option controls the import of
-          the <strong>SYS</strong> (Sensor groups) or the <strong>IX:RTXINITIALIZER</strong>
-          Logix.  Subsequent changes to <strong>Sensor Groups</strong> or <strong>LRoutes
-          </strong> will update the related Logix.  LogixNG will not be aware of those
-          changes. It's recommended to leave this option unchecked.</p>
+          Logix created by other JMRI tools.</p>
+
+          <ul>
+            <li><strong>SYS:</strong> The IX:SYS Logix contains the sensor group defintions.</li>
+            <li><strong>RTX:</strong> These are created by LRoutes and the Route <strong>
+            Export to Logix</strong> option.  This includes the RTXINITIALZER.</li>
+            <li><strong>USS CTC:</strong> These are create by the USS CTC OsIndicator tool.</li>
+            <li><strong>SSL:</strong> These are created to support complex SSL configurations
+            such as Throat-to-Throat.  Since these do not have descriptive system names,
+            they will be imported.  This can lead to unpredictable SSL behavior.  These
+            should be deleted from LogixNG after doing the import.</li>
```

Comment:
If I understand you correctly, SSL Logixs will be imported even if `Import system Logixs also?` is unchecked. If so, I suggest that you move these lines below this section, just before the header "Import Actions".

For me, these points (SYS, RTX, USS, SSL) seems related to the option `Import system Logixs also?` so if SSL isn't affected by that setting, I suggest you move SSL.

Reply:
Will do.